public/pages/SampleData/components/SampleDataBox/SampleDataBox.tsx

```diff
@@ -0,0 +1,113 @@
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import ContentPanel from '../../../../components/ContentPanel/ContentPanel';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+import { PLUGIN_NAME } from '../../../../utils/constants';
+
+interface SampleDataBoxProps {
+  title: string;
+  icon: any;
+  description: string;
+  buttonDescription: string;
+  onLoadData(): void;
+  isLoadingData: boolean;
+  isDataLoaded: boolean;
+  detectorId: string;
+}
+
+export const SampleDataBox = (props: SampleDataBoxProps) => {
+  return (
+    <div style={{ height: 'auto' }}>
+      <ContentPanel
+        title={
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignContent: 'flexStart',
+            }}
+          >
+            {props.icon}
+            <EuiSpacer size="m" />
+            <h2>&nbsp;&nbsp;</h2>
+            <EuiTitle size="s">
+              <h2 style={{ marginTop: '-3px' }}>{props.title}</h2>
+            </EuiTitle>
+          </div>
+        }
+        titleSize="s"
+        badgeLabel={props.isDataLoaded ? 'INSTALLED' : undefined}
+      >
+        <EuiFlexGroup direction="column" gutterSize="l">
+          <EuiFlexItem grow={false} style={{ height: '70px' }}>
+            <p
+              style={{
+                textAlign: 'justify',
+                lineHeight: 1.4,
+                maxHeight: 4.2,
+              }}
+            >
+              {props.description}
+            </p>
+          </EuiFlexItem>
+          <EuiFlexGroup
+            style={{
+              marginTop: '6px',
+              marginBottom: '0px',
+            }}
+            direction="column"
+            alignItems="center"
+          >
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                style={{ width: '300px' }}
+                data-test-subj="loadDataButton"
+                disabled={props.isLoadingData || props.isDataLoaded}
+                isLoading={props.isLoadingData}
+                onClick={() => {
+                  props.onLoadData();
+                }}
+              >
+                {props.isLoadingData
+                  ? 'Creating detector'
+                  : props.isDataLoaded
+                  ? 'Detector created'
+                  : props.buttonDescription}
+              </EuiButton>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              {props.isDataLoaded ? (
+                <EuiLink href={`${PLUGIN_NAME}#/detectors/${props.detectorId}`}>
+                  View detector and sample data
+                </EuiLink>
+              ) : (
+                <EuiLink disabled={true}>&nbsp;</EuiLink>
```

Comment:
Can we just return `null` instead of an disabled `EuiLink`?

Reply:
yes, will change.